c2rust-analyze/src/main.rs

```diff
@@ -272,12 +272,24 @@ fn is_build_script(at_args: &[String]) -> anyhow::Result<bool> {
     Ok(bin_crate_name().is_none() && is_bin_crate(at_args)?)
 }
 
+/// Check whether "no Cargo" mode is enabled.  In this mode, `c2rust-analyze` behaves like `rustc`
+/// instead of like `cargo`.  For example, `C2RUST_ANALYZE_NO_CARGO=1 c2rust-analyze foo.rs` will
+/// process `foo.rs` much like `rustc foo.rs` does.
+fn in_no_cargo_mode() -> bool {
+    env::var_os("C2RUST_ANALYZE_NO_CARGO").is_some()
```

Comment:
Does this also return `true` if the variable is empty? I.e. someone runs the tool as `C2RUST_ANALYZE_NO_CARGO= c2rust-analyze`.

Reply:
Yes - this is consistent with our handling of other env vars like `C2RUST_ANALYZE_TEST_DONT_CATCH_PANIC`